### PR TITLE
skipper: use redis 7

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
https://raw.githubusercontent.com/redis/redis/7.2/00-RELEASENOTES

The https://github.com/zalando/skipper/pull/2895 updates skipper tests to use redis-7

### CODE REVIEW

File: cluster/manifests/skipper/skipper-redis.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: StatefulSet
 metadata:
   labels:
     application: skipper-ingress-redis
-    version: v6.2.7
+    version: "7.2.4"
```
Comment (szuecs) [inline cluster/manifests/skipper/skipper-redis.yaml]: Why not prefix with v?
We do it basically for every container to omit the float issue :)

**Comment by szuecs** [comment]
I don't like that "refactoring" no-op is mixed with image change.
Why do we need to have indentation and version "fixes"?

File: cluster/manifests/skipper/skipper-redis.yaml (hunk 1 above)
Comment (AlexanderYastrebov) [reply cluster/manifests/skipper/skipper-redis.yaml]: I checked other manifests and reverted to use v prefix

**Comment by AlexanderYastrebov** [comment]
> I don't like that "refactoring" no-op is mixed with image change.
> Why do we need to have indentation and version "fixes"?

I thought to make it a valid yaml on the go (by putting template actions into yaml comments) but we can split this change in two so I reverted to version+image change only.

**Comment by RomanZavodskikh** [comment]
:+1:

File: cluster/manifests/skipper/skipper-redis.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: StatefulSet
 metadata:
   labels:
     application: skipper-ingress-redis
-    version: v6.2.7
+    version: v7.2.4
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/skipper-redis.yaml:6]:
```
~$ docker run --rm container-registry.zalando.net/library/redis-7-alpine:7-alpine-20240226 redis-server --version
Redis server v=7.2.4 sha=00000000:0 malloc=jemalloc-5.3.0 bits=64 build=f17de61a0b604ef3
```

**Comment by szuecs** [comment]
:+1:

**Pull request merged**